recorded_future/plugin.spec.yaml

```diff
@@ -9,7 +9,7 @@ status: []
 description: "Recorded Future arms threat analysts, security operators, and incident
   responders to rapidly connect the dots and reveal unknown threats. Using the Recorded Future plugin for Rapid7
 InsightConnect, users can search domain lists, entity lists, and more"
-version: 1.5.5
+version: 1.6.0
```

Comment:
This needs a major version bump since we changed existing variable names. It will now be backward-incompatible